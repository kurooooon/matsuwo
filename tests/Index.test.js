import React from 'react';
import Index from '../pages';
import renderer from 'react-test-renderer';
import { mockAllIsIntersecting } from 'react-intersection-observer/test-utils'

test('<Index> snapshot testing', () => {
  const component = renderer.create(<Index />);
  mockAllIsIntersecting(true);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});