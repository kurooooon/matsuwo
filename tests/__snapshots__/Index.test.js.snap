// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index> snapshot testing 1`] = `
<main>
  <section
    className="css-3mqo5q"
  >
    <h1
      className="css-wu315a"
    >
      matsuwo
    </h1>
  </section>
  <section
    className="css-12a298q"
  >
    <section
      className="css-14un8h8"
    >
      <header
        className="css-1kxox02"
      >
        <h2>
          What's New
        </h2>
      </header>
    </section>
  </section>
  <section
    className="css-1uvcuzn"
  >
    <div>
      <header
        className="css-1kxox02"
      >
        <h2>
          Music
        </h2>
      </header>
      <p>
        <a
          className="css-x54iz"
          href="https://www.youtube.com/channel/UCRSPD9OHzBjDfY9EFE4hDHw"
          target="_blank"
        >
          ...more
        </a>
      </p>
      <header
        className="css-1kxox02"
      >
        <h2>
          Art
        </h2>
      </header>
      <p>
        <a
          className="css-x54iz"
          href="https://www.saatchiart.com/account/profile/218701"
          target="_blank"
        >
          ...more
        </a>
      </p>
    </div>
  </section>
  <section
    className="css-1nzjv6j"
  >
    <div
      className="css-14un8h8"
    >
      <div
        className="css-q4srgx"
      >
        <header
          className="css-792eih"
        >
          <h2>
            About
          </h2>
        </header>
        <div
          className="css-17fiq0y"
        >
          <div
            className="css-1hq4w0s"
          >
            <div
              className="css-f2sjc2"
            >
              <div />
            </div>
          </div>
          <div
            className="css-1qp2ph2"
          >
            <p>
              matsuwo / 松尾竜平
            </p>
            <p>
              北海道札幌での
              <a
                href="https://www.saatchiart.com/account/profile/218701"
                target="_blank"
              >
                アート
              </a>
              の制作や個展、音楽制作やライブ活動を精力的に行い、2019年春より東京での活動をスタート。 2013年matsuwo名義で
              <a
                href="https://tower.jp/artist/2149889/matsuwo"
                target="_blank"
              >
                宅録CD
              </a>
              をリリース。
            </p>
            <p>
              歌詞と曲は心揺さぶられるできごと作品や言葉などありふれた日常の中からインスパイアされ主にJ-popの曲をlogic proやProtoolsで制作しており、音はいたってシンプルで分厚く重ねない。 影響されたアーティストはゆず、ミスターチルドレン、サザンオールスターズ、宇多田ヒカルなどなど。 ライブでは主にアコースティックギターの弾き語り形式で演奏。 使用アコースティックギターはリサイクルショップで購入した1560円のモーリス。 今まで使ってきたどれよりも弾きやすい。
            </p>
            <p>
              アートでは油絵を制作してイタリア、フランス、ニューヨークの展覧会やアートインレジデンスを経験後、音楽ジャケットやチラシのアートワークを担当したりアートフェアや個展などで絵画を発表している。
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="css-141yklu"
  >
    <ul
      className="css-1umhx8z"
    >
      <li>
        <a
          aria-label="ToLo Records Channel / Youtube"
          className="css-wk1r6r"
          href="https://www.youtube.com/channel/UCRSPD9OHzBjDfY9EFE4hDHw"
          onClick={[Function]}
          target="_blank"
        >
          <svg
            alt="ToLo Records Channel / Youtube"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 576 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
            />
          </svg>
        </a>
      </li>
      <li>
        <a
          aria-label="@MATUWOMATUWO / Twitter"
          className="css-wk1r6r"
          href="https://twitter.com/MATUWOMATUWO"
          onClick={[Function]}
          target="_blank"
        >
          <svg
            alt="@MATUWOMATUWO / Twitter"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            />
          </svg>
        </a>
      </li>
      <li>
        <a
          aria-label="Ryuhei Matuso / Facebook"
          className="css-wk1r6r"
          href="https://www.facebook.com/ryuhei.matsuo.50"
          onClick={[Function]}
          target="_blank"
        >
          <svg
            alt="Ryuhei Matuso / Facebook"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
            />
          </svg>
        </a>
      </li>
      <li>
        <a
          aria-label="ryuhei_matsuo / instagram"
          className="css-wk1r6r"
          href="https://www.instagram.com/ryuhei_matsuo/"
          onClick={[Function]}
          target="_blank"
        >
          <svg
            alt="ryuhei_matsuo / instagram"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
            />
          </svg>
        </a>
      </li>
      <li>
        <a
          aria-label="@matumatumatu / tiktok"
          className="css-wk1r6r"
          href="https://www.tiktok.com/@matumatumatu"
          onClick={[Function]}
          target="_blank"
        >
          <svg
            alt="@matumatumatu / tiktok"
            data-file-name="SvgTiktok"
            width="16"
          />
        </a>
      </li>
      <li>
        <a
          aria-label="mail to matsuwo"
          className="css-wk1r6r"
          href="mailto:matsuwo611@gmail.com"
          onClick={[Function]}
          target="_blank"
        >
          <svg
            alt="mail to matsuwo"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"
            />
            <polyline
              points="22,6 12,13 2,6"
            />
          </svg>
        </a>
      </li>
    </ul>
    <ul
      className="css-zmi65u"
    >
      <li>
        © matsuwo / 松尾竜平
      </li>
      <li>
        Design: 
        <a
          href="https://html5up.net/"
        >
          HTML5 UP
        </a>
      </li>
    </ul>
  </section>
</main>
`;
